model/src/main/proto/feedback_reporting.proto

```diff
@@ -288,7 +306,7 @@ message FeedbackReportingDeviceContext {
 }
 
 // Information collected about the current version of the Oppia Android app.
-message FeedbackReportingAppContext {
+message FeedbackReportingOppiaContext {
```

Comment:
I actually thought 'app' was clearer--'Oppia' could mean any number of things: the app, the web frontend, the backend. the name of the tutor, the team name, the organization, etc.

Reply:
Ack, done